### PR TITLE
Cesium3DTileset.disableCollision is removed

### PR DESCRIPTION
**Pull request by furkanaygar**
# Description
In this PR, the already deprecated Cesium3DTileset.disableCollision is removed.

## Issue number and link

#11867

## Testing plan

# Author checklist

- [x] I have submitted a Contributor License Agreement
- [x] I have added my name to `CONTRIBUTORS.md`
- [x] I have updated `CHANGES.md` with a short summary of my change
- [ ] I have added or updated unit tests to ensure consistent code coverage
- [ ] I have update the inline documentation, and included code examples where relevant
- [x] I have performed a self-review of my code

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+### 1.116 - 2024-04-01
+
+#### @cesium/engine
+
+##### Fixes :wrench:
```
Comment (ggetz) [inline CHANGES.md:7]: We typically list these deprecation under `##### Breaking Changes :mega:`
```suggestion
##### Breaking Changes :mega:
```

**Review by ggetz — COMMENTED** [review]
Thanks @furkanaygar!

File: CHANGES.md (hunk 1 above)
Comment (furkanaygar) [reply CHANGES.md]: I fixed it

**Comment by ggetz** [comment]
Thanks @furkanaygar! And congrats on your first PR 🎉

**Pull request merged**